### PR TITLE
Topology Sidebar: keep user-selected alert severity expanded

### PR DESCRIPTION
**Pull request by alfonsomthd**

### CODE REVIEW

File: packages/shared/alert/AlertsPanel.tsx [hunk 1]
```diff
@@ -133,28 +133,38 @@ const AlertAccordionItem: React.FC<AlertAccordionItemProps> = ({
...
   className,
   loaded,
   loadError,
 }) => {
   const { t } = useCustomTranslation();
-  const [expanded, setExpanded] = React.useState('');
+  const [expanded, setExpanded] = React.useState<string>('');
+  let expandedSeverity = expanded;
+  let setExpandedSeverity = setExpanded;
+  if (_.isString(expandedAlertSeverity) && setExpandedAlertSeverity) {
```
Comment (GowthamShanmugam) [inline packages/shared/alert/AlertsPanel.tsx:159]: logic looks good to me but please makesure this line is exceeds rendering

Comment (alfonsomthd) [reply packages/shared/alert/AlertsPanel.tsx:159]: The rendering behavior remains the same.

**Pull request closed without merging**